content/blog/2018-03-23-hydroscoper.md

```diff
@@ -0,0 +1,157 @@
+---
+slug: "hydroscoper"
+title: ὕδωρ + σκοπῶ
```

Comment:
Could you give a hint in english about what the post might be about? To entice people to read it